internal/controller/dispatch_logic.go

```diff
@@ -237,6 +262,18 @@ func aggregateRequests(appWrapper *mcadv1beta1.AppWrapper) Weights {
 	return request
 }
 
+// Aggregate limits
+func aggregateLimits(appWrapper *mcadv1beta1.AppWrapper) Weights {
+	limit := Weights{}
+	for _, r := range appWrapper.Spec.Resources.GenericItems {
+		for _, cpr := range r.CustomPodResources {
+			// TODO: check why not implemented limits spec in CustomPodResource?
+			limit.AddProd(cpr.Replicas, NewWeights(cpr.NotImplemented_Limits))
```

Comment:
Now that limits are relevant, please update the CRD renaming `NotImplemented_Limit` to `Limits`.

Reply:
done